python/python_evaluator.py

```diff
@@ -75,7 +76,7 @@ class ExecArgs(object):
 
     # HALT! do NOT change this without changing corresponding type in the frontend! <----
     # Also note that this uses camelCase because that is standard in JS frontend
-    def __init__(self, evalCode, savedCode="", filePath="", usePreviousVariables=False, *args, **kwargs):
+    def __init__(self, evalCode: str, savedCode="", filePath="", usePreviousVariables=False, *args, **kwargs):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__](https://app.codacy.com/manual/almenon214/AREPL-backend/pullRequest?prid=5181101)